sql/2019/06_Fonts/06_31.sql

```diff
@@ -0,0 +1,29 @@
+#standardSQL
+# 06_31: Popular fonts (see also 02_23)
+CREATE TEMPORARY FUNCTION getFontFamilies(css STRING)
+RETURNS ARRAY<STRING> LANGUAGE js AS '''
+try {
+  var $ = JSON.parse(css);
+  return $.stylesheet.rules.filter(rule => rule.type == 'font-face').map(rule => {
+    var family = rule.declarations && rule.declarations.find(d => d.property == 'font-family');
```

Comment:
> This is a bit pedantic, but in the case where there are multiple font-families supplied in a block, this returns the first occurrence instead of the last (which is what would be applied).

That's true. I'll make a note for the authors to be aware of this, but it's probably so subtle as to not make a difference in the aggregate.

> Will we be combining different versions of fonts like FontAwesome and Font Awesome 5 Free?

I'll leave that for the authors to interpret manually :innocent: